### PR TITLE
Irv unit test

### PR DESCRIPTION
**Pull request by bwyss**
Jasmine Unit Test
================

This PR includes unit tests for the IRV application using Jasmine.
The test coverage is limited to the createRiskIndicator and combineIndicators functions.
For each of the functions tested in this PR, I am providing in the spec sample objects and arrays to be tested against. These samples can be extended in future test development to cover corner cases.
The test coverage for the IRV web application is started in this PR but by no means complete.

Jasmine Overview:
-----------------

**Jasmine in a nutshell:**

Jasmine is an open source testing framework for JavaScript and can run on any JavaScript-enabled platform (It is framework agnostic and does not require a DOM). 
Jasmine allows you to define behaviour specifications (specs) for your software and tests to determine whether your software meets the specification. The whole process is automated. Jasmine follows the principles of Behaviour Driven Development which starts with behavioural specifications as the basis for all your tests.

**Jasmine directory structure:**

A 'jasmine' directory has been added into the 'static' directory allowing the tests to be accessible to a development environment without the need to run a dedicated web server.

lib

 - All the jasmine library dependancies

spec

 - Jasmine javascript spec (test) files
 - A spec contains one or more expectations that test the state of the code
 - An expectation in Jasmine is an assertion that is either true or false
 - A spec with all true expectations is a passing spec
 - A spec with one or more false expectations is a failing spec

src

 - Spec runner html file
 - Spec runner is a collection of Jasmine dependancies
 - Spec runner is also collection of dependancies for the tested application
 - The spec runner includes the Javascript application that is to be tested
 - The spec runner includes the Jasmine spec for a give tested application

To run the Jasmine tests in a development environment, checkout the branch and navigate in a browser to:  YourOQPDevPath/static/jasmine/IRVTests.html

If all the tests are passing you will see:

screen shot 2015-07-31 at 12 38 50 pm

**NOTE** 
There are failure expected without pushing layer from QGIS SVIR toolkit

**TODO**
Provide the CI environment with some SVIR layer that are to be tested

### CODE REVIEW

File: openquakeplatform/openquakeplatform/static/js/irv_viewer.js [hunk 1]
```diff
@@ -105,8 +105,16 @@ function createIndex(la, index) {
     return indicator;
 }
 
-function combineIndicators(nameLookUp, themeObj, JSONthemes) {
```
Comment (nastasi-oq) [inline openquakeplatform/openquakeplatform/static/js/irv_viewer.js]: Normally, an explicit parameter has more priority than an implicit one.
Please, remove testSessionProjectDef from function definition and use arguments[3] and, if present, set it as projectDef inverting the logic.

**Comment by nastasi-oq** [comment]
LGTM
